### ISSUE TITLE
Plugin is incompatible with BepInEx build #647 or newer

### ISSUE THREAD

**Issue by Gravrain** [Silvris/Magicite#2]
After updating my BepInEx installation recently, I noticed that the Magicite plugin failed to load without any errors in the log. After looking into the problem, it seems that BepInEx IL2CPP build #647 changed some package names, meaning any plugins that use them will need to be updated (see [here](https://github.com/BepInEx/BepInEx/issues/521) and [here](https://github.com/BepInEx/BepInEx/issues/464) for details).

The previous build, #577, seems to be the last working version, so I've been using that in the meantime.

**Comment by Silvris** [comment]
Yeah, I've been delaying on updating because Memoria is also still incompatible with the newest BepInEx, and I figured it'd be best to stay compatible.
